cinema/views.py

```diff
@@ -31,8 +41,29 @@
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
-    serializer_class = MovieSerializer
+    queryset = Movie.objects.prefetch_related("genres", "actors")
+
+    @staticmethod
+    def params_to_ints(query_string):
+        return [int(num) for num in query_string.split(",")]
```

Comment:
The `params_to_ints` method in `MovieViewSet` does not handle potential conversion errors. Consider adding error handling to manage cases where non-integer values are provided in the query string, which could otherwise lead to exceptions.